project_2/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.26)
```

Comment:
Not needed for issues https://github.com/Cloud-Storage-Team/algnet/issues/30 and https://github.com/Cloud-Storage-Team/algnet/issues/31.